src/index.html

```diff
@@ -17,16 +18,31 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      class="form-search"
+    >
+      <div class="form-search__image form-search__image_big"></div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <input
+        type="text"
+        data-qa="big"
```

Comment:
The `input` element in the big search form is missing the `data-qa="keypress"` attribute. Please add this attribute to meet the task requirements.